tests/ui/intern-browserstack.js

```diff
@@ -3,6 +3,7 @@
 // packages, suites, excludeInstrumentation, and (if you want functional tests) functionalSuites.
 define(['./_base', './_cli', 'intern'], function(config, cli, intern) {
 
+    // Browsers to run integration testing against.
```

Comment:
We can leave off the version numbers for Safari and Firefox -- they're updated so often that we don't need multiples.